airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -532,7 +532,13 @@ def start_template_dataflow(
 
         :param job_name: The name of the job.
```

Comment:
Is this bug ( #8300) only affecting templates?

Reply:
Yes. It only affects templates.  Native pipelines were not affected by this issue.